practice/pyramid/site.css

```diff
@@ -0,0 +1,22 @@
+.row {
+  box-sizing: border-box;
+  height: 100px;
```

Comment:
Remember my layout problem solving suggestions: set widths from the top down, then heights from the bottom up. The height of each row should be the result of the inner boxes, not set explicitly.